packages/api/tests/endpoints/test-pdrs.js

```diff
@@ -241,3 +289,84 @@ test('DELETE handles the case where the PDR exists in DynamoDb but not in S3', a
   const parsedBody = response.body;
   t.is(parsedBody.detail, 'Record deleted');
 });
+
+test('DELETE removes a PDR from RDS and DynamoDB', async (t) => {
+  // Create the same PDR in Dynamo and PG
+  const newDynamoPdr = fakePdrFactory('completed');
+  const pdrName = newDynamoPdr.pdrName;
+  const newPGPdr = fakePdrRecordFactory({
+    name: pdrName,
+    status: 'completed',
+    collection_cumulus_id: t.context.collectionCumulusId,
+    provider_cumulus_id: t.context.providerCumulusId,
+  });
+
+  // create a new PDR in Dynamo
+  await pdrModel.create(newDynamoPdr);
+
+  // create a new PDR in RDS
+  await pdrPgModel.create(t.context.knex, newPGPdr);
+
+  const response = await request(app)
+    .delete(`/pdrs/${pdrName}`)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`);
+  t.is(response.status, 200);
+
+  // Check Dynamo and RDS. The PDR should have been removed from both.
+  await t.throwsAsync(
```

Comment:
Nit: I believe that the Dynamo models also have an `exists` method. But it's not necessary